# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/moz.build
# Commit: 7a4e3f5f674a
# Full Hash: 7a4e3f5f674ad675fa084bf1f30a79a15e8c7d2a
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-19 04:46:07
# Regressor Bug: 1351231
# File Overlap Count: 2
# Description:
#   Bug 1351231 - PFetch protocol declaration and implementation. r=dom-worker-reviewers,jesup
#   
#   Depends on D138813
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142436
# ==============================================================================

diff -r 9da00c1364a5 -r 7a4e3f5f674a dom/fetch/moz.build
--- a/dom/fetch/moz.build	Wed Jan 18 00:50:19 2023 +0000
+++ b/dom/fetch/moz.build	Wed Jan 18 00:50:19 2023 +0000
@@ -11,9 +11,11 @@
     "BodyExtractor.h",
     "ChannelInfo.h",
     "Fetch.h",
+    "FetchChild.h",
     "FetchDriver.h",
     "FetchIPCTypes.h",
     "FetchObserver.h",
+    "FetchParent.h",
     "FetchService.h",
     "FetchStreamReader.h",
     "FetchStreamUtils.h",
@@ -30,8 +32,10 @@
     "BodyExtractor.cpp",
     "ChannelInfo.cpp",
     "Fetch.cpp",
+    "FetchChild.cpp",
     "FetchDriver.cpp",
     "FetchObserver.cpp",
+    "FetchParent.cpp",
     "FetchService.cpp",
     "FetchStreamReader.cpp",
     "FetchStreamUtils.cpp",
@@ -46,6 +50,7 @@
 
 IPDL_SOURCES += [
     "FetchTypes.ipdlh",
+    "PFetch.ipdl",
 ]
 
 LOCAL_INCLUDES += [